search/indexing_api.py

```diff
@@ -136,15 +136,26 @@
                         'school_name': KEYWORD_TYPE,
                         'school_state_or_territory': KEYWORD_TYPE,
                     }},
+                    'email': KEYWORD_TYPE,
                     'email_optin': BOOL_TYPE,
                     'filled_out': BOOL_TYPE,
                     'first_name': FOLDED_SEARCHABLE_KEYWORD_TYPE,
                     'full_name': FOLDED_SEARCHABLE_KEYWORD_TYPE,
                     'gender': KEYWORD_TYPE,
                     'last_name': FOLDED_SEARCHABLE_KEYWORD_TYPE,
+                    'nationality': KEYWORD_TYPE,
                     'preferred_language': KEYWORD_TYPE,
                     'preferred_name': FOLDED_SEARCHABLE_KEYWORD_TYPE,
                     'pretty_printed_student_id': KEYWORD_TYPE,
+                    'image': KEYWORD_TYPE,
```

Comment:
Nit: suggest moving these up to keep with the lexicographical sort we've been using on key names.